src/_gettsim/parameters/unterhalt.yaml

```diff
@@ -13,49 +13,130 @@ mindestunterhalt:
   unit: Euro
   reference_period: Month
   2016-01-01:
-    6: 335
-    12: 384
-    18: 450
+    1:
+      min_alter: 0
+      max_alter: 5
+      betrag: 335
+    2:
+      min_alter: 6
+      max_alter: 11
+      betrag: 384
+    3:
+      min_alter: 12
+      max_alter: 17
+      betrag: 450
     reference: V. v. 03.12.2015 BGBl. I S. 2188.
   2017-01-01:
-    6: 342
-    12: 393
-    18: 460
+    1:
```

Comment:
You can use "`deviation_from = previous`" here, I think. Then you'll just need to adjust `betrag'.